src/query/service/src/sql/planner/binder/ddl/table.rs

```diff
@@ -363,6 +366,28 @@ impl<'a> Binder {
             )?;
         }
 
+        let (storage_params, path) = match uri_location {
+            Some(uri) => {
+                let uri = UriLocation {
+                    protocol: uri.protocol.clone(),
+                    name: uri.name.clone(),
+                    path: uri.path.clone(),
+                    connection: uri.connection.clone(),
+                };
+                parse_uri_location(&uri)?
+            }
+            None => {
+                let sp = self.ctx.get_storage_params();
```

Comment:
If it's none, we should store it inside the meta?

What if we already migrate the storage data into another storage data?

We must modify the meta of all the fuse tables.